persistence/persistence.go

```diff
@@ -315,6 +316,19 @@ func (a *EstablishedAgreement) GetRetryStartTime() uint64 {
 	return 0
 }
 
+// return the cluster namespace from the TsAndCs of the proposal
+func (a *EstablishedAgreement) GetReqClusterNamespace() string {
```

Comment:
Is it function needed? Will be un-comment in the future?

Reply:
No, I will delete it. Thanks for the catch.